base/channels.jl

```diff
@@ -102,6 +102,8 @@ end
 
 eltype{T}(::Type{Channel{T}}) = T
 
+capacity(c::Channel) = c.sz_max
```

Comment:
should this be `length(Channel)` perhaps? `capacity` is not currently defined anywhere in Base.

will also need to add docs for this.